README.md

```diff
@@ -0,0 +1,36 @@
+Pimcore Bootstrap
+=================
+
+Provides Bootstrap-specific areas for your themes.
+
+Installation
+------------
+
+Using Composer:  
+
+```
+composer require pfaffenbauer/bootstrap
+```
+
+Composer will automatically copy the area files into your `website/var/areas` directory.
+
+Next, you'll have to make the views available. You can either copy the folders inside `views/scripts` into 
+`website/views/scripts` or otherwise:
```

Comment:
It was necessary for me with a relatively clean installation of the most recent release + sample data. But if you know a better way that would be awesome.